Userland/Utilities/shot.cpp

```diff
@@ -35,6 +107,18 @@ int main(int argc, char** argv)
     }
 
     auto app = GUI::Application::construct(argc, argv);
+    auto window = GUI::Window::construct();
+    Optional<Gfx::IntRect> crop_region;
+    if (select_region) {
+        window->set_opacity(0.4);
+        auto& container = window->set_main_widget<SelectableLayover>(window);
+        container.set_fill_with_background_color(true);
+        window->set_fullscreen(true);
+        window->show();
+        app->exec();
+        crop_region = container.region();
```

Comment:
I guess if there is no `crop_region`, the user has aborted and the program should thus exit? Currently, it will then just make a screenshot of the full screen.

Reply:
If there is no `crop_region`, the program will just exit now.